runtime/doc/nvim_python.txt

```diff
@@ -20,16 +20,25 @@ Note: For now only the old Vim 7.3 API is supported.
 ==============================================================================
 2. Quickstart						 *nvim-python-quickstart*
 
+If you used a package manager to install Nvim, there's a good chance that
+it also provides the `neovim` Python package. If it doesn't, follow these
+steps to install the package with Python's package manager pip.
```

Comment:
Maybe backticks around `pip`? Also, a comment before `pip` would be helpful IMO.